src/java.base/share/classes/java/nio/X-Buffer.java.template

```diff
@@ -2173,14 +2190,17 @@ public abstract sealed class $Type$Buffer
      * aligned address.  Specifically, the index should either be decremented by
      * the return value if the latter is not greater than {@code index}, or be
      * incremented by the unit size minus the return value.  Therefore given
-     * <blockquote><pre>
-     * int value = alignmentOffset(index, unitSize)</pre></blockquote>
+     * {@snippet lang=java :
+     *     int value = alignmentOffset(index, unitSize)
+     * }
      * then the identities
-     * <blockquote><pre>
-     * alignmentOffset(index - value, unitSize) == 0, value &le; index</pre></blockquote>
+     * {@snippet lang=java :
+     *     alignmentOffset(index - value, unitSize) == 0, value &le; index
```

Comment:
There is html for less-than, greater-than, etc. in many of the examples that will need be changed too.

Reply:
I don't see any HTML entities that leaked through in this file aside from this one.

Reply:
Hopefully https://github.com/openjdk/jdk/pull/13870/commits/ef684901584386084c88ed291c868c2e8129050b caught them all.

Reply:
Check Files as there are several there.

Reply:
> Check Files as there are several there.

I found one there and one in Path.

Reply:
> > Check Files as there are several there.
> 
> I found one there and one in Path.

Forget it: missed some. Correcting.

Reply:
Re-scanned `java.nio` tree. Hopefully [a531b06](https://github.com/openjdk/jdk/pull/13870/commits/a531b06b6fa2ed34080c64208b2fae4eff450c63) caught everything this time.